### PR TITLE
TC001 - TC003 for TS001

### PR DESCRIPTION
**Pull request by dbmsch**
I've added TC001-TC003 test-cases for test suit TS001

### CODE REVIEW

File: src/SmokeTests/TS001.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package src.SmokeTests;
+
+public class TS001 {
+    // TC001 for AC1 US008
+    // TC002 for AC1 US008
+    // TC003 for AC2 US008
```
Comment (Krislev94) [inline src/SmokeTests/TS001.java:6]: I need a full description please! :)

**Review by Krislev94 — CHANGES_REQUESTED** [review]

File: src/SmokeTests/TS001.java (hunk 1 above)
Comment (dbmsch) [reply src/SmokeTests/TS001.java:6]: I see, I can give you some classes you can understand the java code better

Comment (dbmsch) [reply src/SmokeTests/TS001.java:6]: TC003 is end-to-end test for purchase workflow

Comment (Krislev94) [reply src/SmokeTests/TS001.java:6]: just add one method!

**Review by Krislev94 — APPROVED** [review]
Amazing! you might get a raise this year!

**Pull request merged**